server/src/controllers/Events/resolver.ts

```diff
@@ -273,17 +273,62 @@ ${venue.postal_code} <br>
   }
 
   @Mutation(() => Boolean)
-  async sendEventInvite(@Arg('id', () => Int) id: number) {
+  async sendEventInvite(
+    @Arg('id', () => Int) id: number,
+    @Arg('emailGroups', () => [String], {
+      nullable: true,
+      defaultValue: ['interested'],
```

Comment:
```suggestion
      defaultValue: ['confirmed'],
```
Since the modal defaults to 'confirmed' I think it should be the default here, too.

Reply:
@ojeytonwilliams , Thanks a lot for your feedback. I used default value as **_interested_** because I did not want to break the existing functionality of the emails being sent to the users who have user_chapter_role.interested =true when a new event is created. if we use **_confirmed_** then , since there will be no confirmed users on the new event. notification email will not be sent to any one.

Reply:
Good point, thanks for explaining.  Sorry, I was a bit hasty when I reviewed this.

I (finally) get what you're trying to do and it makes sense. I'll give it a bit more thought over the weekend and get back to you!